### PR TITLE
refactor: Script code

### PR DESCRIPTION
**Pull request by vishal804**
Hello,

Refactor some code and bugs in code 
Please Review 

Thank You 

Link - https://deploy-preview-6--shoppingcorner.netlify.app/

### CODE REVIEW

**Review by ArjunJunna — APPROVED** [review]
Looks fine. You can merge it.

**Review by Anurag021 — APPROVED** [review]
Code looks good to merge.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+  - Ratings
+  - Sort by
+- [Cart-Management](https://shoppingcorner.netlify.app/pages/cart.html)
+- [Wishlist-Management](https://shoppingcorner.netlify.app/pages/wishlist.html)
+- [Authentication](https://shoppingcorner.netlify.app/pages/login.html)
+
+## Technologies Used
+
+- HTML
+- CSS
```
Comment (b0llu) [inline README.md:23]: great work on readme

File: css/product.css [hunk 2]
```diff
@@ -1,5 +1,3 @@
-@import "/css/Utility.css";
-
 .list-heading label {
   margin: 0.5rem 0;
```
Comment (b0llu) [inline css/product.css:2]: you can use px if the value is less than 1rem

File: script.js [hunk 3]
```diff
@@ -1,9 +1,20 @@
...
-};
+});
+
+showConfirmPassword.addEventListener("click", () => {
+  if (confirmPassword.type === "password") {
+    confirmPassword.type = "text";
+  } else {
+    confirmPassword.type = "password";
+  }
+});
```
Comment (b0llu) [inline script.js:20]: great work on the JS file its easily understandable

**Review by b0llu — APPROVED** [review]
Suggested few changes. Everything else looks good

File: css/product.css (hunk 2 above)
Comment (vishal804) [reply css/product.css:2]: Thanks for suggestion using rem Everywhere in project and comfortable with it

File: README.md (hunk 1 above)
Comment (vishal804) [reply README.md:23]: Thank you

File: script.js (hunk 3 above)
Comment (vishal804) [reply script.js:20]: Thank you

**Pull request merged**